pyHS100/antitheft.py

```diff
@@ -0,0 +1,199 @@
+"""
+pyHS100
+Python library supporting TP-Link Smart Plugs/Switches (HS100/HS110/Hs200).
+
+The communication protocol was reverse engineered by Lubomir Stroetmann and
+Tobias Esser in 'Reverse Engineering the TP-Link HS110':
+https://www.softscheck.com/en/reverse-engineering-tp-link-hs110/
+
+This library reuses codes and concepts of the TP-Link WiFi SmartPlug Client
+at https://github.com/softScheck/tplink-smartplug, developed by Lubomir
+Stroetmann which is licensed under the Apache License, Version 2.0.
+
+You may obtain a copy of the license at
+http://www.apache.org/licenses/LICENSE-2.0
+"""
+
+
+class AntiTheft(object):
```

Comment:
Totally off topic. Would you be interested in supporting a python 2 branch? We've dropped the support requirement, but if you're going to support python 2 on a fork, you might as well do it here in this repo to keep all code in one place.